### DISCUSSION TITLE
save pth file size

### DISCUSSION THREAD

**Discussion by Bouhafs-Aek** [open-mmlab/mmaction2#2530] [General]
i need support , the best acc model save is half the epoch model pth file ?  

EXAMPLE  : The best checkpoint   size 200.MB and the epoch size model is :400 MB the double all the time

**Comment by cir7** [comment]
That's reasonable, the best acc model removes the optimizer part, so it's much smaller.

**Reply by Bouhafs-Aek** [reply]
how much the size of the optimizer?

**Reply by cir7** [reply]
it depends on the type of your optimizer, you could load the epoch checkpoint and have a look.

**Reply by Bouhafs-Aek** [reply]
i need to know why the size is double !?  form 200 MB to 400 MB ! , is the checkpoint at epoch is defernet than best-acc model saved ?
